unbxdSearch.js

```diff
@@ -1234,12 +1335,13 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
             }
             this.ajaxCall = jQuery.ajax({
                 url: urlobj.url.replace(this.options.searchQueryParam + "=", "q="),
-                dataType: "jsonp",
+                dataType: "json",
```

Comment:
Please check for the platform, for `COM` we'll get `CORS` error.

Reply:
As mentioned above, this code is to support only new customers.